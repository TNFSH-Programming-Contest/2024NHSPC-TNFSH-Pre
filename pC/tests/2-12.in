88 552
48 21
50 74
30 25
26 75
64 6
5 22
5 33
32 82
66 65
15 83
40 66
23 42
9 71
19 70
4 3
59 50
3 40
3 30
45 84
26 74
27 21
45 23
46 25
72 75
84 39
75 60
8 53
71 37
51 21
10 47
78 32
77 62
19 43
36 50
79 14
44 67
14 78
88 1
65 11
38 83
80 42
31 12
66 72
75 4
59 38
1 4
7 42
84 79
49 44
70 61
67 71
6 29
20 70
15 64
62 78
7 10
37 31
85 63
47 30
42 61
40 25
42 69
59 40
60 12
31 77
7 15
9 66
73 33
13 6
35 74
15 88
85 69
2 32
87 36
16 84
33 79
54 6
14 59
38 3
26 68
29 76
36 7
32 47
45 32
68 7
64 71
67 42
21 32
68 42
5 24
48 24
84 2
55 43
83 51
12 52
73 63
29 85
6 1
88 46
17 80
6 12
65 23
4 32
35 59
63 12
27 38
49 16
33 63
76 75
1 80
35 60
62 80
4 57
53 87
30 44
74 55
5 7
12 34
52 18
43 86
88 56
45 85
2 58
65 41
56 22
27 70
72 22
24 65
55 10
75 18
36 12
22 52
84 44
52 30
65 80
34 15
40 4
12 32
86 71
81 4
48 33
82 54
40 27
64 16
43 77
23 37
74 29
6 38
84 17
54 69
41 84
26 76
35 75
69 80
14 15
54 75
39 45
50 23
31 16
3 29
56 75
4 21
82 69
47 70
15 71
19 36
5 80
1 20
80 12
68 1
1 45
44 15
2 37
34 36
57 16
5 1
67 75
54 40
88 24
22 40
44 27
12 78
12 71
58 53
40 1
34 13
15 3
86 75
72 25
13 73
7 3
80 20
66 62
49 11
58 20
71 82
45 64
37 81
65 5
51 43
10 53
22 21
26 56
3 68
78 30
82 7
40 78
21 54
27 19
19 15
59 25
6 55
30 49
38 39
42 55
21 40
60 54
72 28
28 26
78 4
39 22
4 34
75 21
30 18
77 37
53 23
68 34
20 74
19 61
50 77
19 83
37 48
23 83
43 65
37 40
50 55
72 77
46 44
82 24
39 65
39 44
43 59
27 14
5 78
24 73
76 52
59 74
53 18
67 13
60 18
32 30
43 37
51 19
50 83
40 75
55 63
88 23
67 72
74 40
31 14
58 16
57 80
53 44
10 35
44 75
46 17
5 44
71 73
87 43
23 22
15 5
10 2
81 55
1 36
44 18
1 81
48 71
88 78
9 49
45 20
43 79
83 8
71 62
78 52
83 71
55 30
57 9
40 62
47 60
49 53
59 27
36 76
86 32
17 1
88 37
11 18
18 56
86 87
31 54
85 33
21 29
3 46
63 87
6 8
43 27
57 35
59 17
72 78
41 6
28 21
57 62
81 9
13 35
67 55
33 13
85 4
5 68
19 24
47 14
9 8
79 62
52 19
25 51
5 79
70 56
7 59
14 33
22 17
14 72
88 26
49 14
28 81
73 79
79 19
62 70
75 22
64 24
54 88
81 85
86 9
83 67
87 1
51 3
41 83
31 62
73 76
62 9
25 36
11 16
7 30
84 35
11 4
57 82
88 77
15 6
64 4
28 35
2 12
30 4
63 28
63 18
41 53
36 18
47 79
64 52
58 61
14 40
2 45
39 55
32 69
66 78
5 82
43 18
68 57
26 1
74 6
28 50
18 74
33 22
71 32
39 8
68 79
71 57
56 10
11 6
44 22
66 53
44 24
79 71
22 19
76 6
22 7
76 67
77 87
2 59
46 39
71 38
3 71
81 39
29 69
34 69
43 88
46 10
12 44
85 74
42 48
52 51
28 37
43 29
23 29
42 31
67 78
23 10
27 30
53 63
40 34
73 53
58 7
87 61
20 24
41 46
59 39
1 82
8 28
10 64
74 31
9 25
48 87
62 54
9 76
57 46
13 7
67 81
34 52
22 24
12 5
26 33
68 58
1 72
54 30
70 10
44 58
37 47
70 63
47 69
57 75
47 68
73 66
75 43
84 85
11 59
42 38
26 59
50 84
84 72
71 25
45 73
79 3
18 82
25 33
51 80
20 14
67 82
12 58
5 66
13 50
24 26
10 5
30 72
15 63
1 2
70 29
11 25
75 50
6 5
70 7
15 29
5 62
5 25
9 26
33 77
45 21
72 74
29 65
4 19
30 64
32 63
54 58
50 31
12 57
72 6
15 50
36 3
13 18
66 2
72 65
77 30
20 2
62 12
79 4
47 55
6 65
34 82
21 3
45 18
85 71
28 52
52 73
9 5
50 7
60 72
22 50
64 83
9 59
58 14
39 85
62 24
57 83
46 32
3 1
10 28
73 6
6 86
59 78
14 76
2 13
26 64
40 20
70 50
33 88
86 30
86 45
27 77
44 74
23 20
12 56
2 76
3 34
11 26
16 68
4 13
66 52
30 81
36 57
5 41
64 7
11 73
39 17
83 74
20 51
20 9
14 3
