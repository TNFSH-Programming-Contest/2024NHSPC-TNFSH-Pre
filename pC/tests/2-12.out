53
2 8 11 13 15 16 17 20 21 23 24 25 27 28 29 31 34 35 36 37 38 41 46 47 48 49 51 52 54 55 56 58 60 61 63 64 65 66 67 68 69 70 73 76 77 78 79 80 81 82 85 86 87 